forward/spike.lua

```diff
@@ -4,9 +4,16 @@ local C = require("ffi").C
 local Rewriting = require("rewriting")
 local godefs = require("godefs")
 local IPV4 = require("lib.protocol.ipv4")
+local cmdline = require("cmdline")
+local spike_test = require("spike_test")
 
 local function runmain()
-   if #main.parameters ~= 5 then
+   local params = getparam(main.parameters)
+   if params["t"] then
```

Comment:
I think we should make the test run as a separate script rather than parsing command-line flags.